install.bat

```diff
@@ -0,0 +1,50 @@
+@ECHO off
+
+ECHO "Welcome to the setup for mentorship-backend repo."
+
+ECHO "This installation requires Python version above 3.0."
+
+ECHO "Cloning the repository"
+git clone https://github.com/systers/mentorship-backend.git
+cd mentorship-backend
+
+ECHO "Installing the virtual environment - 'venv'"
+pip3 install virtualenv
+virtualenv venv
+
+CALL venv\Scripts\activate	
+
+ECHO "Activated virtual environment"
+
+ECHO "Installing the required modules"
+pip3 install -r requirements.txt
+
+ECHO "Setting up the environment variables"
+
+SET /P config="Enter value for FLASK_ENVIRONMENT_CONFIG (dev/test/prod)"
+SET FLASK_ENVIRONMENT_CONFIG=%config%
+
+SET /P secret_key="Enter value for SECRET_KEY"
+SET SECRET_KEY=%secret_key%
+
+SET /P security_password_salt="Enter value for SECURITY_PASSWORD_SALT"
+SET SECURITY_PASSWORD_SALT=%security_password_salt%
+
+SET /P mail_sender="Enter value for MAIL_DEFAULT_SENDER "
+SET MAIL_DEFAULT_SENDER=%mail_sender%
+
+SET /P mail_server="Enter value for MAIL_SERVER "
+SET MAIL_SERVER=%mail_server%
+
+SET /P app_mail_username="Enter value for APP_MAIL_USERNAME "
+SET APP_MAIL_USERNAME=%app_mail_username%
+
+SET /P app_mail_password="Enter value for APP_MAIL_PASSWORD "
+SET APP_MAIL_PASSWORD=%app_mail_password%
+
+ECHO "Executing the unit test"
+python3 -m unittest discover tests
```

Comment:
If the tests fail, we should not get a success message as would the next set of lines display! maybe you want to check for errors here